app/data/tasks/fetch_record_csv.py

```diff
@@ -0,0 +1,111 @@
+import csv
+from django.conf import settings
+from django.core.files import File
+from collections import namedtuple
+import tempfile
+import re
+
+from celery import shared_task
+
+from ashlar.models import Record
+from black_spots.models import BlackspotRecordsFile
+
+
+# Keys which cannot be usefully exported to csv
+DROPPED_KEYS = ['media']
+# static record fields
+RECORD_FIELDS = ['record_id', 'created', 'modified', 'occurred_from',
+                 'occurred_to', 'lat', 'lon', 'location_text',
+                 'city', 'city_district', 'county', 'neighborhood', 'road',
+                 'state', 'weather', 'light']
+
+
+def FIELD_TRANFORMS():
+    def date_iso(d):
+        return d.isoformat()
+
+    FieldTransform = namedtuple('FieldTransform', ['field', 'transform'])
+    return {
+        'record_id': FieldTransform(field='uuid', transform=lambda uuid: str(uuid)),
+        'created': FieldTransform(field='created', transform=date_iso),
+        'modified': FieldTransform(field='modified', transform=date_iso),
+        'occurred_from': FieldTransform(field='occurred_from', transform=date_iso),
+        'occurred_to': FieldTransform(field='occurred_to', transform=date_iso),
+        'lat': FieldTransform(field='geom', transform=lambda geom: geom.y),
+        'lon': FieldTransform(field='geom', transform=lambda geom: geom.x),
+    }
+
+
+@shared_task
+def export_records(occurred_min, occurred_max, record_label=settings.PRIMARY_RECORD_LABEL):
```

Comment:
Just to re-emphasize, I think this needs to be a `record_type_id` (not label); `RecordType.label` is not guaranteed to be unique (and in fact often isn't -- by default, running the event import script multiple times will create RecordTypes with duplicate labels).